fuzz/fuzz_targets/router_target.rs

```diff
@@ -107,9 +110,10 @@ pub fn do_test(data: &[u8]) {
 	}
 
 	let logger: Arc<Logger> = Arc::new(test_logger::TestLogger{});
+	let chain_monitor = Arc::new(chaininterface::ChainWatchInterfaceUtil::new(Network::Bitcoin, Arc::clone(&logger)));
```

Comment:
Hmm, to keep the fuzzer from choking on the complication can you create a dummy chain watcher that reads from a byte from the input and just returns something based on that?

Reply:
I've it written but can't pass InputData to chain_monitor due to lifetime conflicting requirements, hmmm

Reply:
You have to do something like the InputData struct in full_stack_target, but should be able to do so without the Vec input copy.

Reply:
Done, fixup squash at 2d87bad